src/v0/destinations/braze/transform.js

```diff
@@ -236,6 +246,10 @@ function processTrackWithUserAttributes(message, destination, mappingJson, proce
       );
       if (dedupedAttributePayload) {
         requestJson.attributes = [dedupedAttributePayload];
+      } else if (reqMetadata?.features && reqMetadata.features['filter-code']) {
```

Comment:
Use constant `FEATURE_FILTER_CODE` instead of hard coded string `filter-code` everywhere.

Reply:
Updated